src/coin-selection/lib/Convex/CoinSelection.hs

```diff
@@ -243,7 +245,7 @@ balanceTransactionBody
   -- TODO: If there are any stake pool unregistration certificates in the transaction
   -- then we need to provide a @Map StakeCredential Lovelace@ here.
   -- See https://github.com/input-output-hk/cardano-api/commit/d23f964d311282b1950b2fd840bcc57ae40a0998
-  let unregPoolStakeBalance = mempty
+  let unregPoolStakeBalance = unregBalance protocolParams txbodycontent2
```

Comment:
Should remove todo? Is the current implementation complete?

Reply:
Yep I believe that TODO is implemented now